src/Sulu/Bundle/AudienceTargetingBundle/Admin/AudienceTargetingAdmin.php

```diff
@@ -142,7 +149,7 @@ public function configureViews(ViewCollection $viewCollection): void
     public function getSecurityContexts()
     {
         return [
-            'Sulu' => [
+            $this->suluSecuritySystem => [
```

Comment:
I think we are mixing here something. The parameter used by the security system is I think not the parameter which should be used here. So I think it is correct that this was hardcoded before.

Reply:
I don't think we are mixing anything here. If you change the other parameter, and the user can't login aftewards, that's clearly a bug, so it is definitely not correct that it is currently hardcoded. I don't know why somebody would want to change that, but if they do it should continue to work. And this seems like the proper fix to me. And I would accept the BC break for that.

But as already said, I'd be also fine with deprecating that parameter, recommend to use the `system` tag in the webspace instead, live with the fact that it currently does not work, and completely remove it with the next major release.

Reply:
If we go on the parameter its a behaviour change on website context. It will set the permission registered in every admin also to the new security system. But the permission defined in the Admin classes are only for the Sulu security system not the website context security system which the user did define.

Reply:
Since the admin class should only be defined to be registered in the `admin` context, this should not be a problem. However, as already said, I think the easiest way forward is simply deprecating the configuration value and keep the behavior as it is. I don't think that many people changed that value in the admin configuration anyway.